exonum/src/runtime/rust/mod.rs

```diff
@@ -12,7 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Built-in Rust runtime module.
+//! The runtime for the native Rust services.
```

Comment:
```suggestion
//! The current runtime is for running native services written in Rust.
```

Reply:
I'm not sure about this. Why is the original line bad?

Reply:
Because I'm not sure that original line is grammatically correct 😄 @Maria-Nosyk help with it.